src/Matrix.cpp

```diff
@@ -283,6 +325,21 @@ FP_PRECISION* Matrix::getA() {
 
 
 /**
+ * @brief Get the A component (excluding the diagonal) of the CSR form of the
```

Comment:
Perhaps it might be nice to include a `@details` comment block describing what the `A` component physically represents?